### PR TITLE
copy edit README

### PR DESCRIPTION
**Pull request by apotheon**
I noticed a typo in the README while telling someone else about how much I like bdd-for-c, and went on a bit of a copy-edit rampage.

I hope you find this useful.  Please let me know if you have any concerns about my changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,7 +127,7 @@ spec("some feature") {
     }
 
     after() {
-        printf("%i tests run. %i failed.\n", test_count, tests_failed);
+        printf("%i tests run.  %i failed.\n", test_count, tests_failed);
```
Comment (grassator) [inline README.md]: Are two spaces intentional here?

File: README.md [hunk 2]
```diff
@@ -2,17 +2,18 @@
 
 The `bdd-for-c` library is a [BDD][bdd] test framework for the [C language][c].
 
+
 ## Quick Start
 
 To start, simply download the framework header file which can be easily done
-with curl on Linux and OS X:
+with curl on [MacOS][mac], [Linux][linux], and [BSD Unix][bsd]:
```
Comment (grassator) [inline README.md]: Not sure it makes sense to link to distributions here. Maybe more helpful would be to tell people how to save it from the browser?

**Comment by grassator** [comment]
@apotheon Thank you for your effort. In general looks great, just had a couple of questions / comments to look at.

File: README.md (hunk 1 above)
Comment (apotheon) [reply README.md]: I'm a "two spaces between sentences" kind of person.  If you're not so much, that's fine.  I don't recall whether I did that automatically or intentionally, off the top of my head, though.

File: README.md (hunk 2 above)
Comment (apotheon) [reply README.md]: I guess I'm not sure what you mean about "link to distributions".  Are you saying you don't think the links to the different OSes are useful, or are you saying something about the URI/link for curl?

I'm not really wedded to the OS links, so getting rid of those (or keeping them) is fine with me.  I might have been a little overly documentation-oriented when I made these edits, like back in the day when I wrote fourteen articles a month for TechRepublic.  Your call on keeping or discarding the links (obviously).

I think explaining how to save the file from the browser *in addition to* explaining how to get it with curl makes sense, though.

**Comment by apotheon** [comment]
Let me know what you think of my replies to your comments.  Sorry it took me a couple days to get back to this.

Some time this weekend, I'll probably add some instructions for people to get bdd-for-c via browser in addition to curl download, and add that to the PR.

File: README.md (hunk 2 above)
Comment (grassator) [reply README.md]: Yes, sorry I meant that links to OSes are probably not useful as they won't really help identified what needs to be done and in general I would expect people to know which OS they are running.

And yes adding browser download instructions would be great, but I could also do it after merge the PR myself.

File: README.md (hunk 1 above)
Comment (grassator) [reply README.md]: This is not really common I would say and I'm just afraid that I would get another PR from someone else putting it back to one space :) If you don't mind I would stick to defaults.

**Comment by apotheon** [comment]
I think I addressed everything except writing a "download from the browser" section.  If I get to it before you, I'll open a new pull request if you want to merge this now, I guess.

**Comment by grassator** [comment]
Merged this for now. Thank you again for the effort!

**Pull request merged**